dex-template/__DexName__.ts.template

```diff
@@ -92,6 +94,14 @@ export class __DexName__
     return null;
   }
 
+  // Returns estimated gas cost of calldata for this DEX in multiSwap
```

Comment:
Maybe need to point in comment that it makes sense to change only if `Dex` supports `Optimism`?

Reply:
I think we can have it correctly for all new DEX (not just Optimism or Arbitrum ones), since I already did them all the existing ones.